Scripts/script-MD2HTML.yml

```diff
@@ -0,0 +1,29 @@
+commonfields:
+  id: MD2HTML
+  version: -1
+name: MD2HTML
+script: |-
+  from markdown2 import Markdown
+
+  markdowner = Markdown(extras=["tables"])
+  file_entry = demisto.args().get('file_id', '')
+  file_path = demisto.getFilePath(file_entry)['path']
+  demisto.results(file_path)
+  fopen = open(str(file_path), "r")
```

Comment:
Better handle IO exception in case file does not actually open

Reply:
You are also not closing the file that you are reading, try using: with open(str(filepath), “r”) as file: ...